packages/terra-content-container/src/ContentContainer.module.scss

```diff
@@ -39,9 +42,25 @@
     }
   }
 
-  .focusoncontainer {
+  .setpadding {
```

Comment:
this is a bit vague, can we rename this to: `content-container-focused-padding`

Reply:
Updated